instrumentation/grpc-1.6/testing/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6/AbstractGrpcTest.java

```diff
@@ -863,11 +863,17 @@ static class ErrorProvider implements ArgumentsProvider {
     public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
       return Stream.of(
           arguments(Status.UNKNOWN.withCause(new RuntimeException("some error"))),
-          arguments(Status.PERMISSION_DENIED.withCause(new RuntimeException("some error"))),
+          arguments(Status.DEADLINE_EXCEEDED.withCause(new RuntimeException("some error"))),
           arguments(Status.UNIMPLEMENTED.withCause(new RuntimeException("some error"))),
+          arguments(Status.INTERNAL.withCause(new RuntimeException("some error"))),
+          arguments(Status.UNAVAILABLE.withCause(new RuntimeException("some error"))),
+          arguments(Status.DATA_LOSS.withCause(new RuntimeException("some error"))),
           arguments(Status.UNKNOWN.withDescription("some description")),
-          arguments(Status.PERMISSION_DENIED.withDescription("some description")),
-          arguments(Status.UNIMPLEMENTED.withDescription("some description")));
+          arguments(Status.DEADLINE_EXCEEDED.withDescription("some description")),
+          arguments(Status.UNIMPLEMENTED.withDescription("some description")),
+          arguments(Status.INTERNAL.withDescription("some description")),
+          arguments(Status.UNAVAILABLE.withDescription("some description")),
+          arguments(Status.DATA_LOSS.withDescription("some description")));
     }
```

Comment:
Nit: it'd be nice to have a test that checks client errors that are not server errors (e.g. not found)

Reply:
Added `NOT_FOUND` to the test. Note that there are 2 tests `errorReturned` and `errorThrown`, `NOT_FOUND` is not with error status in `errorReturned` but is in `errorThrown`. That is because in `errorThrown` the actual grpc status is `UNKNOWN`, maybe the way this test throws the exception is not the correct way to set grpc status.